incubator/virtualcluster/pkg/util/mccontroller/helper.go

```diff
@@ -0,0 +1,133 @@
+/*
```

Comment:
Is it possible to remove "CR" from all variable and method names?

It is a good abstraction for both build-in and CR objs and having a "CR" substring in the names looks a little confusing.